libs/feature/search/src/lib/metadata-view/metadata-view.module.ts

```diff
@@ -6,17 +6,33 @@ import { StoreModule } from '@ngrx/store'
 import { MdViewEffects } from './state/mdview.effects'
 import { MdViewFacade } from './state/mdview.facade'
 import { MD_VIEW_FEATURE_STATE_KEY, reducer } from './state/mdview.reducer'
-import { DataTableComponent } from './data-table/data-table.component'
+import { RecordMetadataComponent } from './record-metadata/record-metadata.component'
+import { DataViewMapComponent } from './data-view-map/data-view-map.component'
+import { DataViewTableComponent } from './data-view-table/data-view-table.component'
+import { RecordExportsComponent } from './record-exports/record-exports.component'
+import { RecordApisComponent } from './record-apis/record-apis.component'
 
 @NgModule({
-  declarations: [DataTableComponent],
+  declarations: [
+    RecordMetadataComponent,
+    DataViewMapComponent,
+    DataViewTableComponent,
+    RecordExportsComponent,
```

Comment:
I think exports and apis are related to a dataset, not a record

Reply:
I agree, would you be ok with `data-exports` and `data-apis` @fgravin, @jahow ?

Reply:
well I think you went with these names [because of me](https://github.com/geonetwork/geonetwork-ui/pull/157#issuecomment-926733874) so... I guess whichever is fine :)